src/Azure.Functions.Cli/Helpers/SecurityHelpers.cs

```diff
@@ -11,35 +11,11 @@
 using System.Text;
 using System.Threading.Tasks;
 using System.Security.Cryptography;
-using System.Linq;
-using System.Reflection;
 
 namespace Azure.Functions.Cli.Helpers
 {
     internal static class SecurityHelpers
     {
-        private static bool CheckCurrentUser(dynamic obj)
```

Comment:
So what this just a bunch of unused methods?

Reply:
Yes. I think in previous iterations of the internal action they were used, but that action has been disabled since quite a while ago